### PR TITLE
Cleanup of Recent Doc Updates

### PR DESCRIPTION
**Pull request by shannona**
A bit of clarity, a bit of punctuation, a bit of format standardization.

### CODE REVIEW

**Review by Fonta1n3 — COMMENTED** [review]
`⚠️ **The following functionality is only available on Electrum if you build the master branch from source. Hopefully it will be avilable for download soon. You can still create the multisig wallets on Electrum, but you won't be able to easily pass PSBTs from Electrum to any third party app.** ⚠️` 

This is now outdated, the user needs Electrum 4.0.0 which can be downloaded [here](https://download.electrum.org/4.0.0b1/)

File: Docs/Electrum-guide.md [hunk 1]
```diff
@@ -1,48 +1,56 @@
...
 ![](../Images/Electrum/10_exportVpub.png)
 
 ### 10. Enter cosigner #2
 
 ![](../Images/Electrum/11_enter_cosigner2.png)
 
-### 11. Come back to FN2 and export your 1st public key descriptor which holds all the xpubs you need. You can get this by going to the wallets tab, enabling the Electrum wallet, and tapping the "info" button to export the devices seed and public key descriptors.
+### 11. Come back to FN2 and export your first public key descriptor, which holds all the xpubs you need
+
+You can get this by going to the wallets tab, enabling the Electrum wallet, and tapping the "info" button to export the device's seed and public key descriptors.
```
Comment (Fonta1n3) [inline Docs/Electrum-guide.md:53]: The info button no longer exists, no big deal but we should go through and update these guides to match the latest UI.

**Comment by Fonta1n3** [comment]
+1 LGTM, we will need to add some updated images and minor corrections to perfectly match the latest UI. Thanks @shannona

**Review by ChristopherA — APPROVED** [review]
+1 LGTM. I'm going to leave it to @Fonta1n3 to address those items he brought up himself in future PRs to this master. If you put doc changes into develop branch, make sure they are separate so you can cherry pick them.

Any remaining items that you'd like @shannona to do that you don't fix this week, please prepare a list of them for Shannon by next Tuesday the 7th.

**Pull request merged**